lib/realtime/vehicle.ex

```diff
@@ -102,15 +120,16 @@ defmodule Realtime.Vehicle do
 
     trip_id = VehiclePosition.trip_id(vehicle_position)
     trip = trip_fn.(trip_id)
-    headsign = trip && trip.headsign
+    headsign = VehiclePosition.headsign(vehicle_position) || (trip && trip.headsign)
```

Comment:
Where does swiftly get its vehicle headsign? Is it just from the trip? Is there any situation where we'd prefer swiftly's vehicle headsign or prefer the GTFS trip headsign?